internal/node/_node_bin/node

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
```

Comment:
oops don't check in this file! (we should fix it so we either stop writing it in the srcs or gitignore it)